src/components/chatroom/header/header.chatroom.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { useState } from "react";
+import searchIcon from "./mag.png";
+
+type HeaderProps = {
+  onSearchChange: React.ChangeEventHandler<HTMLInputElement>;
+};
+
+function Header({ onSearchChange }: HeaderProps) {
+  let [searchToggle, setSearchToggle] = useState(false);
+
+  return (
+    <div className="Header">
+      <h1>채팅</h1>
+      {searchToggle === true ? (
+        <input placeholder="검색" onChange={onSearchChange} />
+      ) : null}
```

Comment:
`searchToggle`이 `boolean` 타입이라 
`null`을 리턴하는 것보다는 `false`를 리턴하는 게 맞지 않을까 싶습니다...! 

```suggestion
      {searchToggle && (
        <input placeholder="검색" onChange={onSearchChange} />}
```
아니면 이렇게 더 간단하게 써 볼 수도 있을 것 같습니다!